client/src/main/java/org/apache/rocketmq/client/impl/producer/DefaultMQProducerImpl.java

```diff
@@ -1326,10 +1326,12 @@ public TransactionSendResult sendMessageInTransaction(final Message msg,
                     }
 
                     if (localTransactionState != LocalTransactionState.COMMIT_MESSAGE) {
-                        log.info("executeLocalTransactionBranch return: {} message: {}", localTransactionState, msg);
+                        log.info("executeLocalTransactionBranch return: {} messageTopic: {} transactionId: {}",
+                            localTransactionState, msg.getTopic(), msg.getTransactionId());
```

Comment:
IMO message tag and key should be logged.

Reply:
done